packages/components/src/modal/README.md

```diff
@@ -97,6 +97,14 @@ If this property is added, it will determine whether the modal requests to close
 - Required: No
 - Default: true
 
+### showCloseIcon
```

Comment:
Am I right in assuming that the "close button" is not the only way we can close a modal? What about "Esc" key. I'm wondering if we should rename this prop `isDismissible` to match the `Notice` API and make it so that we can't close the modal with any method.

Reply:
isDismissable sounds better, thanks. good point about escape, i'll test that and block if needed (you can't close the modal - its blocking)

Reply:
changed to isDismissable. in my testing, the escape key does not close the modal, we should add that feature in a separate PR.